ui/lib/kv/addon/components/kv-list-filter.hbs

```diff
@@ -1,35 +1,14 @@
-<div class="navigate-filter">
-  <div class="field" data-test-nav-input>
-    <p class="control has-icons-left">
-      <Input
-        id="secret-filter"
-        class="filter input"
-        placeholder="Filter secrets"
-        @value={{@filterValue}}
-        @type="text"
-        data-test-component="kv-list-filter"
-        {{on "input" this.handleInput}}
-        {{on "keydown" this.handleKeyDown}}
-        {{on "focus" this.setFilterIsFocused}}
-        {{did-insert this.focusInput}}
-        autocomplete="off"
-      />
-      <Icon @name="search" class="search-icon has-text-grey-light" />
-    </p>
-  </div>
-</div>
-{{#if (and this.filterIsFocused this.isFilterMatch)}}
-  <p class="help has-text-grey is-size-8 has-left-padding-xs" data-test-help-enter>
-    <kbd>ENTER</kbd>
-    to go to see details.
-    <kbd>ESC</kbd>
-    to clear input.
-  </p>
-{{else}}
-  <p class="help has-text-grey is-size-8 has-left-padding-xs" data-test-help-tab>
-    <kbd>TAB</kbd>
-    to autocomplete.
-    <kbd>ESC</kbd>
-    to clear input.
-  </p>
-{{/if}}
\ No newline at end of file
+<form {{on "submit" (perform this.handleSearch)}}>
+  <Hds::SegmentedGroup as |S|>
```

Comment:
@hellobontempo can probably speak more to this, but should we make an hds ticket to keep track of this component being used? 🙂